Add missing meta descriptions (#235)
mongodb/docs-php-library: source/reference/method/MongoDBCollection-replaceOne.txt

MongoDB\\Collection::replaceOne()
=================================

.. meta::

>    :description: Use `replaceOne()` to replace a single document in a collection based on filter criteria, with options for upsert, collation, and more.

Reviewer: ```suggestion
   :description: Replace a single document in a collection based on filter criteria with the MongoDB PHP Library, with options for upsert, collation, and more.
```
